### DISCUSSION TITLE
Is it maintained?

### DISCUSSION THREAD

**Discussion by alexey-sh** [gram-js/gramjs#719] [General]
There are 20 open pull requests, and some of them are ready to be merged. 
So it looks like no one cares about community support. Is that so?

**Comment by painor** [comment]
Eh not really. I merge some of them that are essential or break something. I don't check github that much anymore and most of the progress happens on the telegram chat :/

**Reply by alexey-sh** [reply]
thanks for the comment
may I ask you to share the tg chat?

**Reply by rojvv** [reply]
https://t.me/gramjschat
